chart/assets/operations/instance_groups/locket.yaml

```diff
@@ -0,0 +1,38 @@
+# Disable tuning /proc/sys kernel parameters as locket is running on containers.
+- type: replace
+  path: /instance_groups/name=locket/jobs/name=locket/properties/set_kernel_parameters?
+  value: false
+
+# Add quarks properties for locket.
+- type: replace
+  path: /instance_groups/name=locket/jobs/name=locket/properties/quarks?
+  value:
+    ports:
+    - name: locket
+      protocol: TCP
+      internal: 8891
+    run:
+      healthcheck:
+        locket:
+          readiness:
+            exec:
+              command:
+                - "/usr/bin/nc"
+                - "-z"
+                - "127.0.0.1"
+                - "8891"
```

Comment:
Should probably use a TCP probe instead, but I don't think that's covered under this PR.

Reply:
> Should probably use a TCP probe instead

@andrew-edgar Can you take a look at this?

I'll try to get this PR through CI as-is (because it is a revert of a revert), and will merge if it passes, so any changes should be in a separate PR.